docs/event-streams.md

```diff
@@ -0,0 +1,186 @@
+
+Change topics:
+
+```bash
+# Single topic
+docker run \
+-e KAFKA_BROKERS=kafka:9092 \
+-e KAFKA_SOURCE_TOPICS=my-topic \
+-e PROXY_KAFKA_RESPONSE_TOPICS=my-proxy-topic \
+accenture/reactive-interaction-gateway
+
+# Multiple topics
+docker run \
+-e KAFKA_BROKERS=kafka:9092 \
+-e KAFKA_SOURCE_TOPICS=my-topic1,my-topic2 \
+-e PROXY_KAFKA_RESPONSE_TOPICS=my-proxy-topic1,my-proxy-topic2 \
+accenture/reactive-interaction-gateway
+```
+
+In addition to topics you can configure also consumer group ID.
+> __NOTE:__ same group ID will be used for all topics.
+
+Change group ID:
+
+```bash
+docker run \
+-e KAFKA_BROKERS=kafka:9092 \
+-e KAFKA_SOURCE_TOPICS=my-topic \
+-e PROXY_KAFKA_RESPONSE_TOPICS=my-proxy-topic \
+-e KAFKA_GROUP_ID=my-group-id \
+accenture/reactive-interaction-gateway
+```
+
+### Change producer topics
+
+Rig can produce "monitoring" events when HTTP endpoint is called in proxy and as a target of HTTP endpoint itself, see [API Gateway docs](./api-gateway#sync).
+
+Producing "monitoring" events is by default using `rig-request-log` topic.
+
+Change topic:
+
+```bash
+docker run \
+-e KAFKA_BROKERS=kafka:9092 \
+-e REQUEST_LOG=console,kafka \
+-e KAFKA_LOG_TOPIC=my-log-topic \
+accenture/reactive-interaction-gateway
+```
+
+Producing events from proxy as a target of a request is by default disabled.
+
+Change topic:
+
+```bash
+docker run \
+-e KAFKA_BROKERS=kafka:9092 \
+-e PROXY_KAFKA_REQUEST_TOPIC=my-proxy-request-topic \
+accenture/reactive-interaction-gateway
+```
+
+### SSL
+
+SSL for Kafka is by default disabled. To enable it and set certificates use following setup:
+
+```bash
+docker run \
+-e KAFKA_BROKERS=kafka:9092 \
+-e KAFKA_SSL_ENABLED=1 \
+-e KAFKA_SSL_KEYFILE_PASS=abcdefgh \
+accenture/reactive-interaction-gateway
+
+# Change default paths for certificates
+docker run \
+-e KAFKA_BROKERS=kafka:9092 \
+-e KAFKA_SSL_ENABLED=1 \
+-e KAFKA_SSL_KEYFILE_PASS=abcdefgh \
+-e KAFKA_SSL_CA_CERTFILE=my.ca.crt.pem \
+-e KAFKA_SSL_CERTFILE=my.crt.pem \
+-e KAFKA_SSL_KEYFILE=my.key.pem \
+accenture/reactive-interaction-gateway
+```
+
+### SASL
+
+SASL for Kafka is by default disabled. To enable it and set credentials use following setup:
+
+```bash
+docker run \
+-e KAFKA_BROKERS=kafka:9092 \
+-e KAFKA_SASL=plain:myusername:mypassword \
+accenture/reactive-interaction-gateway
+```
+
+## Kinesis
+
+### Enable Kinesis
+
+Similar as Kafka, Kinesis is by default disabled and can be enabled via `KINESIS_ENABLED` environment variable.
+
+```bash
+# Kinesis disabled
+docker run accenture/reactive-interaction-gateway
+
+# Kinesis enabled
+docker run -e KINESIS_ENABLED=1 accenture/reactive-interaction-gateway
+
+# Configure AWS region
+docker run -e KINESIS_ENABLED=1 -e KINESIS_AWS_REGION=eu-west-3 accenture/reactive-interaction-gateway
```

Comment:
wouldn't you need to use the aws docker-tag?

Reply:
True, good point!